lte/protos/oai/nas_state.proto

```diff
@@ -304,15 +304,33 @@ message UeNetworkCapability {
   uint32 uea = 3;
   uint32 ucs2 = 4;
   uint32 uia = 5;
-  uint32 spare = 6;
-  uint32 csfb = 7;
-  uint32 lpp = 8;
-  uint32 lcs = 9;
-  uint32 srvcc = 10;
-  uint32 nf = 11;
-
-  bool umts_present = 20;
-  bool misc_present = 21;
+ uint32 prosedd = 6;
```

Comment:
Instead of replacing the existing fields in the protobuf, please add all new fields with new field ids, else this will cause conflict with state written using old proto definitions.